src/components/icon/icon.stories.mdx

```diff
@@ -3,11 +3,11 @@ import basicDemo from './demo/basic.twig';
 import inlineDemo from './demo/inline.twig';
 import './icon.scss';
 
-<Meta title="Objects/Icon" />
+<Meta title="Components/Icon" />
 
 # Icon Object
```

Comment:
Should this be changed to `Icon` instead of `Icon Object`? Same for table

Reply:
✅